pkgs/by-name/go/gokey/package.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+let
+  version = "0.1.2-unstalbe-2023-11-16";
+  pname = "gokey";
+in buildGoModule {
+  inherit pname version;
+  src = fetchFromGitHub {
+    owner = "cloudflare";
+    repo = pname;
```

Comment:
usage of `pname` is considered anti pattern https://github.com/NixOS/nixpkgs/issues/277994

Reply:
so you want me to hardcode the string?

Reply:
yes, I should have made it more clear